home.css

```diff
@@ -0,0 +1,645 @@
+
+  .hamburger {
+    display: none;
+  }
+
+  .navbar li {
+    margin-bottom: 0;
+    font-family: var(--lato);
+    font-size: 14px;
+    font-weight: 300;
+  }
+
+  .navbar a {
+    color: var(--darkgray);
+  }
+
+  .navbar {
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    gap: 30px;
+  }
+
+  #logo {
+    margin-left: 0;
+    margin-right: 0;
+  }
+
+  .nav-menu {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    position: inherit;
+    gap: 16px;
+    background-color: transparent;
+    font-family: 'Poppins', sans-serif;
+    font-size: 16px;
+  }
+
+  .nav-item {
+    display: block;
+    align-content: center;
+    padding: 0;
+    margin-right: 0;
+    border-bottom: none;
+  }
+
+  #nav-buy-tickets {
+    padding: 10px;
+    background-color: var(--orange);
+    margin-left: auto;
+  }
+
+  /* HEADLINE */
+
+  .headline {
+    background-image: url(images/main_big.png);
+    background-position: left top;
+    background-size: 100%;
+  }
+
+  .title h1 {
+    font-size: 50px;
+  }
+
+  .title h2 {
+    font-size: 30px;
+  }
+
+  .title h3 {
+    font-size: 30px;
+  }
+
+  .description {
+    font-size: 16px;
+    line-height: 1.9;
+  }
+
+  .title-type-1 {
+    font-size: 28px;
+  }
+
+  .title-type-2 {
+    font-size: 28px;
+  }
+
+  /* PROGRAM */
+
+  .program-features-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    gap: 2%;
+  }
+
+  .program-feature {
+    display: flex;
+    flex-direction: column;
+    width: 220px;
+    height: 220px;
+  }
+
+  .program-feature-title {
+    font-size: 18px;
+  }
+
+  .program-feature-description {
+    font-size: 14px;
+  }
+
+  /* ARTISTS */
+
+  .artist-all-container {
+    grid-template-columns: auto auto;
+    gap: 50px;
+  }
+
+  .artist-container {
+    height: 120px;
+  }
+
+  .artist-img {
+    min-width: 120px;
+    min-height: 120px;
+  }
+
+  .artist-img img {
+    box-shadow: 3px 3px 3px 2px rgba(0, 0, 0, 0.2);
+  }
+
+  .artist-info {
+    margin-left: 18px;
+  }
+
+  .artist-name {
+    font-size: 18px;
+  }
+
+  .artist-work {
+    font-size: 16px;
+  }
+
+  .artist-p {
+    font-size: 12px;
+  }
+}
+
+/* ABOUT */
+
+.about .headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-items: center;
+  text-align: center;
+  margin-top: 75px;
+}
+
+.about .description {
+  margin: 30px auto;
+}
+
+/* LOGO */
+
+.logo-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.about .logo-description {
+  margin: 30px auto;
+  font-family: var(--lato);
+  font-size: 12px;
+  font-weight: 300;
+  line-height: 1.3rem;
+  min-width: 260px;
+  width: 60%;
+}
+
+.logo-section img {
+  width: 50%;
+}
+
+/* PREVIOUS CONCERTS */
+
+.previous-concerts {
+  text-align: center;
+}
+
+.concerts-img-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 40px;
+}
```

Comment:
- [ ] The images in this container has to be in a row. Currently you have them stacked one on top of the other as a result of `flex-direction: column`. But you should have instead `flex-direction: row` on larger screen devices. 🙏
### Your Design
![image](https://user-images.githubusercontent.com/46839548/169112249-5283c7f9-db01-47ab-87df-bd709841a957.png)

### Design Guideline
![image](https://user-images.githubusercontent.com/46839548/169111514-7a54acae-9127-4a8e-a289-f9b947b0a324.png)